.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,24 @@
+name: 'ShellCheck'
+
+on: 
+  push:
+    branches: ["master"]
+  pull_request:
+
+jobs:
+  shellcheck:
+    name: shellcheck
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Run ShellCheck
+      uses: ludeeus/action-shellcheck@master
```

Comment:
There is also a tagged release if you think it would be safer

Reply:
I actually think `master` here is probably just fine, if there is ever an issue or something weird going on, we can disable the action or look at it at that time.